openwhisk/openwhisk.yaml

```diff
@@ -0,0 +1,100 @@
+# Task to define Runtime Parameters
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+    name: openwhisk
+spec:
+    inputs:
+        resources:
+            - name: runtime-git
+              type: git
+        params:
+            - name: TARGET_IMAGE_NAME
+              description: name of the image to be tagged and pushed
+            - name: TARGET_IMAGE_TAG
+              description: tag the image before pushing
+              default: "latest"
+            - name: DOCKERFILE
+              description: The path to the dockerfile to build
```

Comment:
This is the path within the input git resource, correct?

Reply:
yes, path within Runtime Repo, updated description to specify the same.